tools/regionalgam/.shed.yml

```diff
@@ -0,0 +1,90 @@
+categories:
+    - Ecology
+description: Implementation of a method for compution abundance.
+long_description: Implementation of a method for computing an abundance index with the regional GAM approach, an extension of the two-stages model introduced by Dennis et al. (2013). This index offers the best performance for a wide range of sampling frequency, providing greater robustness and unbiased estimates compared to the popular linear interpolation approach (Schmucki et al. 2015).
+
+homepage_url: https://github.com/RetoSchmucki/regionalGAM 
+name: regionalgam
+owner: ecology 
+remote_repository_url: https://github.com/galaxyecology/tools-ecology/tools/regionalgam
+type: unrestricted
+
+repositories:
+  regionalgam_ab-index:
```

Comment:
Is this also needed in this repo? I know it was needed in the other repo, but we should really avoid this. Think about it this way. Even if testing turns green and everything works perfectly, if this list is not up-to-date the TS installation will be broken and we have no way to test this atm :(

Reply:
Ok I try a tool suite, Thank for your comments